.github/workflows/links.yaml

```
@@ -0,0 +1,48 @@
+# Adapted from UCL ARC's Python Tooling repository
+# https://github.com/UCL-ARC/python-tooling/
+
+# This is a basic workflow to check for broken links with GitHub Actions
+
+name: Link Checker
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'  # Every Sunday at midnight UTC
+
+jobs:
+  link-checker:
+    timeout-minutes: 2
+    runs-on: ubuntu-latest
```

```suggestion
    if: github.event.pull_request.draft == false
    runs-on: ubuntu-latest
```
Probably worth specifying this shouldn't run on draft PRs?